backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HandlingUnitsBL.java

```diff
@@ -1118,6 +1119,7 @@ public void setClearanceStatusRecursively(
 
 		hu.setClearanceStatus(clearanceStatusInfo.getClearanceStatus().getCode());
 		hu.setClearanceNote(clearanceStatusInfo.getClearanceNote());
+		hu.setClearanceDate(TimeUtil.asTimestamp(clearanceStatusInfo.getClearanceDate()));
```

Comment:
i think in such a case it's very risky to not take the timezone (org into account).
Please either make sure to have the OrgId and use OrgDAO.getTimezone,
Or probably even better, use InstantAndOrgId